### DISCUSSION TITLE
Will GPUI get support for some more advanced window Options?

### DISCUSSION THREAD

**Discussion by a-usr** [zed-industries/zed#15611] [Feature Requests]
I was considering to build my own wayland desktop shell using gpui, but upon investigating I noticed that thats currently not possible due to the fact that I dont have any way of controlling the layer that gpui windows are drawn on (For reference see the [zwlr_layer_shell documentation](https://wayland.app/protocols/wlr-layer-shell-unstable-v1), similar concepts also exist on other platforms). 
Aside from that I also can't seem to control the position of a window, be it through anchor points or absolute choordinates, or wether it should be ignored by the window manager / compositor (on linux and possibly mac?)

So the question is: Are there any plans for expanding on that?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
